### DISCUSSION TITLE
ImageUploader上传组件使用自定义上传后就不显示自带的loading了吗？

### DISCUSSION THREAD

**Discussion by xiaohua001** [ant-design/ant-design-mobile#6326] [Q&A]
ImageUploader上传组件使用自定义上传后就不显示自带的loading了吗？
**背景：**
想使用app 自己原生提供的上传能力（已参考官网示例实现了功能），故调用自己的相机，但是想用UI 组件自带的“上传中“的效果(暂未实现)，自己做的全局的loading 会影响图，在弱网状态下，我想选了图片后 让它自己上传，不耽误我继续选下一张（或者选下几张）的操作。

**Comment by openapphub** [comment]
2025 同问，有解决方案吗
